### ISSUE TITLE
Can support for proxy services be increased?

### ISSUE THREAD

**Issue by scwcd** [64bit/async-openai#59]
Hi,

Can support for proxy services be increased? It's like this:

```
async fn execute<O>(&self, request: reqwest::Request) -> Result<O, OpenAIError>
    where
        O: DeserializeOwned,
    {
        let client = match self.proxy {
            Some(p) => reqwest::Client::builder().proxy(reqwest::Proxy::all(p)?).build()?,
            None => reqwest::Client::new(),
        }; 
         ...
}
```

**Comment by 64bit** [comment]
Hi @scwcd

You're in luck! @TmLev added a feature in #58  where you can provide custom `reqwest` client.
Its released in v0.9.5

**Comment by 64bit** [comment]
Closing this as it is resolved.
